string direction_right
string direction_left
int8 speed_left
int8 speed_right